Marlin/ultralcd.cpp

```diff
@@ -1552,7 +1552,8 @@ void kill_screen(const char* lcd_msg) {
         lcd_synchronize();
       }
 
-    #elif ENABLED(PROBE_MANUALLY)
+    #endif
+    #if ENABLED(PROBE_MANUALLY)
```

Comment:
I'm thinking this should at least be…
```cpp
#if ENABLED(LCD_BED_LEVELING) && ENABLED(PROBE_MANUALLY)
```
Since we don't want to add anything without `LCD_BED_LEVELING`.